### PR TITLE
[Wasm] Implemented TryEnterFullScreenMode and ExitFullScreenMode

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #2526 

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?

TryEnterFullScreenMode and ExitFullScreenMode are not implemented on WASM.

## What is the new behavior?

TryEnterFullScreenMode and ExitFullScreenMode are implemented on WASM.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/FullScreenModeTests.xaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿<UserControl
+    x:Class="UITests.Shared.Windows_UI_ViewManagement.FullScreenModeTests"
```
Comment (agneszitte-nventive) [inline src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/FullScreenModeTests.xaml]: Indentation / format ;)

Comment (MartinZikmund) [reply src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/FullScreenModeTests.xaml]: Done :-)

File: src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿namespace Windows.UI.ViewManagement {
+
+	export class ApplicationView {
+
+		public static setFullScreenMode(turnOn: boolean) {
+			if (turnOn) {
+				document.documentElement.requestFullscreen();
```
Comment (carldebilly) [inline src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: You should use https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenEnabled to determine if it's available before opening it (and return `false` to `TryEnterFullScreenMode()`

Comment (MartinZikmund) [reply src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: Done :-)

File: src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿namespace Windows.UI.ViewManagement {
```
Comment (carldebilly) [inline src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: Codacy Issue found: ['namespace' and 'module' are disallowed](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4878148)

File: src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs [hunk 4]
```diff
@@ -36,5 +39,16 @@ public string Title
...
 			}
 		}
+
+		public bool TryEnterFullScreenMode() => SetFullScreenMode(true);
+
+		public void ExitFullScreenMode() => SetFullScreenMode(false);
+
+		private bool SetFullScreenMode(bool turnOn)
+		{
+			var jsEval = $"{ApplicationViewTsType}.setFullScreenMode({turnOn.ToString().ToLowerInvariant()})";
```
Comment (carldebilly) [inline src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs]: Codacy Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4878148)

File: src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts (hunk 3 above)
Comment (MartinZikmund) [reply src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: I'm not sure what to do with this one 🤔 , `namespace` are used almost in all `.ts` files - like https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Wasm/ts/Windows/Devices/Geolocation/Geolocator.ts or https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Wasm/ts/Windows/Storage/StorageFolder.ts

File: src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs (hunk 4 above)
Comment (MartinZikmund) [reply src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs]: Fixed

File: src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts (hunk 3 above)
Comment (carldebilly) [inline src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: Codacy Issue found: ['namespace' and 'module' are disallowed](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4878148)

Comment (MartinZikmund) [reply src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: I still don't know how to resolve this one 🤔

Comment (jeromelaban) [reply src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: Don't mind those.

Comment (carldebilly) [reply src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts]: Still trying to tune Codacy.  Sorry, still false positives for typescript code.

**Pull request merged**